app/controllers/concerns/question_and_answer_grouping_helper.rb

```diff
@@ -22,7 +22,7 @@ def self.objects_without_method(array, method)
   def self.all_objects(objects, method, objects_without_method)
     objects_with_method = objects - objects_without_method
 
-    objects_with_method.sort_by { |item| item.try(method) }
+    objects_with_method = objects_with_method.sort_by { |item| item.try(method) }
```

Comment:
You could change this to use `sort_by!` then you wouldn't need to add the assign because it is destructive